cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -166,12 +166,24 @@ set(CMAKE_SYSTEM_INCLUDE_PATH "${EMSCRIPTEN_ROOT_PATH}/system/include")
 # but if these are adjusted, ${CMAKE_ROOT}/Modules/CheckIncludeFile.cmake will fail, since it depends on being able to compile output files with predefined names.
 #SET(CMAKE_LINK_LIBRARY_SUFFIX "")
 #SET(CMAKE_STATIC_LIBRARY_PREFIX "")
-#SET(CMAKE_STATIC_LIBRARY_SUFFIX ".bc")
```

Comment:
Oh, `#` is a comment sign in CMake, these lines had been commented out. (but left for later reference) The default option has been CMake's default, which is `.a`.